tests/console/rabbitmq.pm

```diff
@@ -17,14 +17,13 @@ use base "consoletest";
 use strict;
 use warnings;
 use testapi;
-use utils 'systemctl';
+use utils;
 
 sub run {
     select_console 'root-console';
-    assert_script_run('zypper -n in rabbitmq-server');
+    zypper_call 'in rabbitmq-server python-pika wget';
     systemctl 'start rabbitmq-server';
     systemctl 'status rabbitmq-server';
-    assert_script_run('zypper -n in python-pika wget');
```

Comment:
please keep the separate call. "rabbitmq-server" is the service we want to test, "python-pika" and "wget" are merely test requirements hence the later install